### ISSUE TITLE
Tracing span enter/exit events are logged at the info level

### ISSUE THREAD

**Issue by florv** [seanmonstar/warp#670]
When a logging implementation for the `log` crate is initialized (such as `env_logger`), tracing span enter and exit events end up being logged at the info level. Thus, during normal usage of an application, many “-> Server::run” and “<- Server::run” messages are produced at the info level which does not appear to be particularly friendly.

The issue can be reproduced with:

~~~toml
[dependencies]
env_logger = "0.7"
log = "0.4"
warp = "0.2.4"
~~~

and

~~~rust
use warp::Filter;

#[tokio::main]
async fn main() {
    env_logger::init();
    let hello = warp::path::end()
        .map(|| "Hello world!");
    warp::serve(hello)
        .run(([127, 0, 0, 1], 3030))
        .await;
}
~~~

which produces the following when run with log printed on stderr:

~~~
$ RUST_LOG=info cargo run
    Finished dev [unoptimized + debuginfo] target(s) in 0.19s
     Running `target/debug/warp_span_log`
[2020-07-22T14:40:37Z INFO  warp::server] Server::run; addr=V4(127.0.0.1:3030)
[2020-07-22T14:40:37Z INFO  warp::server] listening on http://127.0.0.1:3030 
[2020-07-22T14:40:37Z INFO  warp::server] -> Server::run
[2020-07-22T14:40:37Z INFO  warp::server] <- Server::run
[2020-07-22T14:40:51Z INFO  warp::server] -> Server::run
[2020-07-22T14:40:51Z INFO  warp::server] <- Server::run
…
~~~

**Comment by jxs** [comment]
this is now fixed on the latest `tracing` crate, to update it run: `cargo update -p tracing`.
thanks!

**Comment by apiraino** [comment]
For future reference, the fix mentioned can be traced in [tracing 0.1.17](https://github.com/tokio-rs/tracing/releases/tag/tracing-0.1.17) (If I am not wrong)
